resources/migrations/000_migrations.yaml

```diff
@@ -3670,3 +3670,68 @@ databaseChangeLog:
                   name: result_metadata
                   type: text
                   remarks: 'Serialized JSON containing metadata about the result columns from running the query.'
+  - changeSet:
+      id: 58
+      author: senior
+      changes:
+        - createTable:
+            tableName: dimensions
+            remarks: 'Stores references to alternate views of existing fields, such as remapping an integer to a description, like an enum'
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: field_id
+                  type: int
+                  remarks: 'ID of the field this dimension row applies to'
+                  constraints:
+                    deferrable: false
+                    foreignKeyName: fk_dimension_ref_field_id
+                    initiallyDeferred: false
+                    nullable: false
+                    references: metabase_field(id)
+                    deleteCascade: true
```

Comment:
not sure if you need it or not but just a heads up if you want an index on your FKs you'll have to add it manually